Void/Void.Core/Commands.fs

```diff
@@ -3,10 +3,13 @@
 [<RequireQualifiedAccess>]
 type Command =
     | Noop
+    | PublishEvent of Event
+    //
+    | ChangeToMode of Mode

```

Comment:
Absolutely. I was blindly reciting the "entities are things and functions are actions rule."